### PR TITLE
[#165887235] Introduce fiscal code fac-simile on Profile Screen

### PR DESCRIPTION
**Pull request by valentinacala**
This pr introduces a new component which renders a fac-simile of the fiscal code.

Full component: 
image

Landscape mode component:
image

**The component in a generic screen**
*iOS X*
CardSidesPreview

*iOS SE*
FCpreviewSE

*Android*
FCpreviewAndroid

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Please include a Pivotal story at the beginning of the PR title (see below).
  </td>
    </tr>
  </tbody>
</table>



  Example of PR titles that include pivotal stories:

  * single story: `[#123456] my PR title`
  * multiple stories: `[#123456,#123457,#123458] my PR title`

    
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: ts/@types/react-native-barcode-builder.d.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+declare module 'react-native-barcode-builder';
```
Comment (fpersico) [inline ts/@types/react-native-barcode-builder.d.ts]: Pleas add a proper definition here.

File: ts/components/FiscalCodeComponent.tsx [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+              : [styles.fullText, styles.fulleNameText]
+          ]}
+        >
+          {profile.name.toUpperCase()}
+        </Text>
+      </React.Fragment>
+    );
+  }
+
+  // TODO: add real barcode
```
Comment (fpersico) [inline ts/components/FiscalCodeComponent.tsx]: Old stuff?

Comment (valentinacala) [reply ts/components/FiscalCodeComponent.tsx]: yes, removed with https://github.com/teamdigitale/io-app/pull/1139/commits/b0b1eee1be12e0483b8ce42ff6e692acbac53f35

File: ts/@types/react-native-barcode-builder.d.ts (hunk 1 above)
Comment (valentinacala) [reply ts/@types/react-native-barcode-builder.d.ts]: the file is updated with https://github.com/teamdigitale/io-app/pull/1139/commits/d8937d609376870c8e8408125fa03291e1145389

File: ts/@types/react-native-barcode-builder.d.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
+declare module 'react-native-barcode-builder' {
+    import { PureComponent } from "react";
+
+    export interface BarcodeProps {
+        value: string;
+        width?: number;
+        height?: number;
+        background?: string;
+        format: string;
```
Comment (fpersico) [inline ts/@types/react-native-barcode-builder.d.ts]: Format is not a simple string

Comment (valentinacala) [reply ts/@types/react-native-barcode-builder.d.ts]: A more detailed type definition is introduced within https://github.com/teamdigitale/io-app/pull/1139/commits/36d1d0ae27f47b9b94e7271fea218c93396a6016

**Pull request merged**